### PR TITLE
Rack Manager/Rack Manager Facts Plugins, Testcases and Examples

### PR DESCRIPTION
**Pull request by alisha-k-kalladassery**
### Description
Rack Manager/Rack Manager Facts Plugins, Testcases and Examples

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes sanity testing.
  - [ ] All sanity tests pass. (`$ ansible-test sanity`).
  - [ ] All unit tests pass.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: roles/oneview_rack_manager_facts/LICENSE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright (2016-2020) Hewlett Packard Enterprise Development LP
```
Comment (AvijitOS) [inline roles/oneview_rack_manager_facts/LICENSE:1]: i think we need to change the license year to 2022

File: plugins/modules/oneview_rack_manager.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    password: my_password
+    api_version: 4400
+    state: present
+    data:
+        hostname : "172.18.6.15"
+        username : "username"
+        password : "password"
+        force: false
+  delegate_to: localhost
+  when: contents.rack_manager.variant == 'DL'
```
Comment (AvijitOS) [inline plugins/modules/oneview_rack_manager.py:78]: is this necessary? i mean how many types of variants supported now

File: roles/oneview_rack_manager/LICENSE [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Copyright (2016-2020) Hewlett Packard Enterprise Development LP
```
Comment (AvijitOS) [inline roles/oneview_rack_manager/LICENSE:1]: change the copyright year to 2022

File: plugins/modules/oneview_rack_manager.py [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+extends_documentation_fragment:
+    - hpe.oneview.oneview
+    - hpe.oneview.oneview.params
+    - hpe.oneview.oneview.validateetag
+'''
+
+EXAMPLES = '''
+- name: Add a Rack Manager
+  oneview_rack_manager:
+    hostname: 172.16.101.48
```
Comment (AvijitOS) [inline plugins/modules/oneview_rack_manager.py:67]: Don't put any ip address, keep place holder

**Review by AvijitOS — APPROVED** [review]

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**